apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -399,12 +403,34 @@ const Portfolio = ({ selectedWalletOption }: PortfolioProps) => {
     trackEvent('Portfolio - User refreshed balances');
   }, [accountService]);
 
+  const onRowClick = React.useCallback(
+    (index: number) => {
+      const token = portfolioBalances[index].tokens[0].token;
+      pushToHistory(`/token/${token.chainId}-${token.address}`);
+    },
+    [portfolioBalances, pushToHistory]
+  );
+
+  const tableContext = React.useMemo<Context>(
+    () => ({
+      intl,
+      showBalances,
+      hoveredRow,
+      rowContext: {
+        onClick: onRowClick,
+        setHovered: setHoveredRow,
```

Comment:
I think this should be handled directly by the component, I get the we mostly do it for the chevron animation, but we could do that by using styles and pointing to this component when the main row gets hovered on the virtualized table

Reply:
Answered above!